docs/specification/STU1.md

```diff
@@ -312,11 +312,24 @@ Authorization: Bearer i8hweunweunweofiwweoijewiwe
 
 ## Request Context Change
 
-Similar to the Hub's notifications to the subscriber, the subscriber MAY request context changes with an HTTP POST to the `hub.topic` url. The Hub MUST either accept this context change by responding with any successful HTTP status or reject it by responding with any 4xx or 5xx HTTP status. The subscriber MUST be capable of gracefully handling a rejected context request. 
+Similar to the Hub's notifications to the subscriber, the subscriber MAY request context changes with an HTTP POST to the `hub.topic` url. The Hub SHALL either accept this context change by responding with any successful HTTP status or reject it by responding with any 4xx or 5xx HTTP status. The subscriber SHALL be capable of gracefully handling a rejected context request. 
 
-Once a requested context change is accepted, the Hub MUST broadcast the context notification to all subscribers, including the original requestor. 
+Once a requested context change is accepted, the Hub SHALL broadcast the context notification to all subscribers, including the original requestor. 
 
 ### Request Context Change Request
+###### Request Context Change Parameters
+Field | Optionality | Type | Description
+--- | --- | --- | ---
+`timestamp` | Required | *string* | ISO 8601-2 timestamp in UTC describing the time at which the event occurred with subsecond accuracy. 
+`id` | Required | *string* | Event identifier used to recognize retried notifications. This id SHALL be uniquely generated by the subscriber and could be a GUID.
```

Comment:
```suggestion
`id` | Required | *string* | Event identifier used to recognize retried notifications. This id SHALL be uniquely generated by the subscriber and could be a GUID. Following an accepted context change request, the hub MAY re-use this value in the broadcasted event notifications.
```

@wmaethner - I think that we need this additional sentence to fully address #170.